rts/lra/test/pom.xml

```diff
@@ -232,6 +231,10 @@
                 </goals>
                 <phase>pre-integration-test</phase>
                 <configuration>
+		  <java-opts>
+		    <java-opt>${server.jvm.object.store.arg}</java-opt>
+		    <!--<java-opt>-agentlib:jdwp=transport=dt_socket,server=y,suspend=y,address=5005</java-opt>-->
```

Comment:
I don't think this line should be in the JBTM-3607 commit, it could even be removed

Reply:
Maybe, instead of removing it completely, we could introduce a property to debug WildFly on-demand. For example, a property named `<wildfly-debug>` with a comment like "This property can be used to debug WildFly when it is appended to WildFly's JVM's arguments. Common use cases are: `maven-wildfly-plugin` and `arquillian's WildFly container`"

Reply:
I'll create a JIRA for it

Reply:
https://issues.redhat.com/browse/JBTM-3619